src/style-config/buttons.scss

```diff
@@ -28,6 +28,7 @@ values are [ background/border color, hover, active, text ]
 $buttonColorSets: (
  ("blue", $blue, $azul, $blue-bright, $white),
  ("magenta", $magenta, $magenta-dull, $magenta-light, $white),
+ ("magenta-dull", $magenta-dull, $magenta-dull, $magenta-light, $white),
```

Comment:
There'll be no hover style here, I'd ask UX about this @harpalsingh

Reply:
Maybe we could use `darken`?

Reply:
The new DLS work actually using a 20% black color on top of the base color to imply "darker", so not the darken() sass mixin but rather mix(black, $magenta-dull, 20), since darken does not directly only add black.

Reply:
Should we use that in this case?

Reply:
Yea, since we have no other set color, we should do that, as they will all be updated in time to use that formula